semantic-python/test/fixtures/1-03-empty-tuple.py

```diff
@@ -1,3 +1,3 @@
 # CHECK-JQ: .scope == {}
-# CHECK-TREE: (#unit)
+# CHECK-TREE: #record{}
```

Comment:
This test was broken and I didn’t even know it!

Reply:
😮